exampleSite/content/dev/list/_index/index.md

```diff
@@ -0,0 +1,22 @@
++++
+headless = true
```

Comment:
Interesting to combine headless and the content fragment. We didn't do that before. What is the reasoning behind that?

Reply:
Since it's in the dev directory, it's not as important to keep it simple and doing it this way would keep a directory that may become complex a bit cleaner. Since dev subdirectories can contain any sort of tests then it would be important to keep them as understandable, easy to create and easy to change as possible.

Reply:
Ok basically just an optimization for ease of use for the dev section. Good